integration_files/SweRV_EH1/sim.py

```diff
@@ -312,6 +312,31 @@ def cp_compiled_test(test, bin_dir, idx):
 	cp_program_hex_cmd 	= "cp %s %s/program.hex" % (program_hex, _CORE)
 	os.system(cp_program_hex_cmd)
 
+# def cp_vcd(test, bin_dir, idx):
```

Comment:
Why is this all commented?

Reply:
Removed these lines. These are not used now.

Reply:
pls make a commit in the same branch with the lines removes